CONFIGURING_ANALYZERS.md

```diff
@@ -0,0 +1,134 @@
+# Configuring Analyzers
+
+There are many other analyzers included in Lucene and there are occasions where custom analyzers are needed. 
+There is now support for configuring additional analyzers without needing to further modify couchdb-lucene. 
+This extension is a loose adaptation of the [eXist-db/Lucene integration](https://github.com/eXist-db/exist/blob/develop/extensions/indexes/lucene/src/org/exist/indexing/lucene/AnalyzerConfig.java)
```

Comment:
thanks for mentioning this, we need to be very sure this isn't a derivative work as that exist-db code is licensed under LGPL and this project is not.